R/support_panel.R

```diff
@@ -103,10 +103,9 @@ support_panel <- function(
         "This dashboard is a new service that we are developing. If you
               have any feedback or suggestions for improvements, please submit
               them using our ",
-        shiny::tags$a(
+        dfeshiny::external_link(
           href = form_url,
-          "feedback form",
-          .noWS = c("after")
+          link_text = "feedback form"
```

Comment:
Want a full stop at the end of this line

Reply:
Done